src/main/java/pl/javastart/task/Processor.java

```diff
@@ -0,0 +1,31 @@
+package pl.javastart.task;
+
+public class Processor extends Component implements Overclockable {
+    public static final int MAX_TEMPERATURE = 100;
+    int timing;
+    int currentTemperature = 50;
+
+    public Processor(String model, String producent, String seriesNumber, int timing) {
+        super(model, producent, seriesNumber);
+        this.timing = timing;
+    }
+
+    @Override
+    public void overlock() {
+        try {
+            checkTemperature();
+            timing += MAX_TEMPERATURE;
+            currentTemperature += 10;
```

Comment:
Unikaj stosowania tzw. "magicznych liczb". Zastąp ją dobrze nazwaną stałą https://youtu.be/eY1li8xwX3M